src/components/pages/admin/events/dashboard/fanNotifications/Index.js

```diff
@@ -233,15 +240,27 @@ class Index extends Component {
 			};
 		}
 
-		updateNotification
+		let resolveSent = false;
+
+		const sent = updateNotification
 			? this.updateNotification(broadcastData)
 			: this.createNotification(broadcastData);
+
+		const cast = Promise.resolve(sent);
+
+		cast.then(function(v) {
+			resolveSent = true;
+		}, function(e) {
+			console.error(e);
+		});
+
+		resolveSent && this.gradualTimer();
```

Comment:
resolveSent will likely not be true at the end of this function.
You'll need to make this function async
```suggestion
		let resolveSent = false;

		updateNotification
			? await this.updateNotification(broadcastData)
			: await this.createNotification(broadcastData);
		this.gradualTimer();
```